woocommerce/woocommerce-android
# [STP] Address design feedback

Closes: #9035


### Description

Two issues were found in the proposed scan to pay UI:
* The dialog used to show QR doesn't have rounded corners
* Button should have the white color of the text in both dark and  light mode (this is the app's global issue)

p1683630224493639-slack-C0354HSNUJH

### Testing instructions

Do both in dark and light mode
* Run STP flow and notice rounded corners on the dialog
* Notice that the "colored" buttons in the app now have white font both in dark and light mode

### Images/gif

<img width="300" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/9bd1d2eb-4509-4664-978a-69bfb9597bcb">
<img width="300" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/8a4d31aa-570f-4c02-aa8b-099d257c1f77">
<img width="300" alt="image" src="https://github.com/woocommerce/woocommerce-android/assets/4923871/48839d7e-b389-4ac5-bb19-ef80d6ed437c">
